### PR TITLE
Schema

### PR DESCRIPTION
**Pull request by jburel**
Minor changes to match changes in https://github.com/openmicroscopy/bioformats/pull/1632
- Version is 2 for ome.xsd
- Check of Documentation for ND Filter

cc @hflynn

### CODE REVIEW

File: formats/schemas/january-2015.txt [hunk 1]
```diff
@@ -13,7 +13,8 @@ correctly using this new release. Some files that failed to validate
 before will now be valid. It is important to update any file readers and
 writers to understand the changes.
 
-The version number of all schema files is now 1.
+The version number of all schema files is now 1, expected ome.xsd which
```
Comment (hflynn) [inline formats/schemas/january-2015.txt]: expected -> except

File: formats/schemas/january-2015.txt [hunk 2]
```diff
@@ -56,6 +57,7 @@ OME
 
 - Expanded documentation for ``AcquisitionDate`` describing supported
   precision.
+- Update documentation for ``NDFilter`` to reflect usage
```
Comment (hflynn) [inline formats/schemas/january-2015.txt]: Update -> Updated

**Comment by hflynn** [comment]
Two minor nitpicks, should be fine once they are fixed

**Comment by hflynn** [comment]
Looks fine on staging, merging and releasing - thanks!

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**